projects/cypress/CY8CKIT_064S0S2_4343W/mtb/aws_demos/afr.mk

```diff
@@ -181,7 +181,8 @@ SOURCES+=\
 	$(wildcard $(CY_AFR_ROOT)/demos/coreMQTT/*.c)\
```

Comment:
Thank you for updating this Cypress Makefile. You will also need to update:
[vendors\cypress\WICED_SDK\apps\demo\aws_demo\aws_demo.mk](https://github.com/aws/amazon-freertos/blob/master/vendors/cypress/WICED_SDK/apps/demo/aws_demo/aws_demo.mk)
[vendors\cypress\WICED_SDK\libraries\aws\aws.mk](https://github.com/aws/amazon-freertos/blob/master/vendors/cypress/WICED_SDK/libraries/aws/aws.mk)

Reply:
please check this [commit](https://github.com/aws/amazon-freertos/pull/2537/commits/09f117053a734f47e94418c012902c28fdc3cfe2).